Taskfile.yml

```diff
@@ -456,12 +456,13 @@ tasks:
       - 'if [ -z "${DOCKER_REGISTRY}" ]; then echo "Error: DOCKER_REGISTRY must be set"; exit 1; fi'
       - az acr repository update --name ${DOCKER_REGISTRY} --image "{{.DOCKER_PUSH_TARGET}}/{{.CONTROLLER_DOCKER_IMAGE}}" --write-enabled false --delete-enabled false
     vars:
+      # This strips the leftmost .*.azurecr.io/ from the PUSH_TARGET as the acr repository update command doesn't want that bit in the image parameter
       DOCKER_PUSH_TARGET:
         sh: |
           if [ ! -z "${DOCKER_PUSH_TARGET}" ]; then
-            echo "${DOCKER_PUSH_TARGET}"
+            echo "${DOCKER_PUSH_TARGET}" | sed -e "s|^.*.azurecr.io/||"
```

Comment:
az cli automatically strips it out, I guess?

Reply:
That's being applied to the `--name ${DOCKER_REGISTRY}` argument, _not_ the `--image` argument (I've tested this locally)